packages/atomic/src/pages/examples/ipx.html

```diff
@@ -136,24 +136,16 @@
         inset: unset;
       }
 
-      atomic-search-interface:not(.atomic-search-interface-search-executed)
-        > atomic-ipx-modal
+      :is(atomic-search-interface:not(.atomic-search-interface-search-executed), atomic-search-interface:not(.atomic-search-interface-search-executed))
+        > :is(atomic-ipx-modal, atomic-ipx-embedded)
         > div[slot='body']
-        > atomic-layout-section,
-      atomic-search-interface:not(.atomic-search-interface-search-executed)
-        > atomic-ipx-embedded
-        > div[slot='body']
-        > atomic-layout-section {
+        > atomic-layout-section
+        > :is(atomic-result-list, atomic-query-summary) {
```

Comment:
This is the local dev html. The fix needs to be in the `atomic.ipx.layout.mustache` as well